warp/handlers/stats.go

```diff
@@ -0,0 +1,40 @@
+// (c) 2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package handlers
+
+import (
+	"time"
+
+	"github.com/ava-labs/subnet-evm/metrics"
+)
+
+type handlerStats struct {
+	// SignatureRequestHandler metrics
+	signatureRequest        metrics.Counter
+	signatureHit            metrics.Counter
+	signatureMiss           metrics.Counter
+	signatureProcessingTime metrics.Gauge
+}
+
+func newStats() *handlerStats {
+	return &handlerStats{
+		signatureRequest:        metrics.GetOrRegisterCounter("signature_request_count", nil),
+		signatureHit:            metrics.GetOrRegisterCounter("signature_request_hit", nil),
+		signatureMiss:           metrics.GetOrRegisterCounter("signature_request_miss", nil),
+		signatureProcessingTime: metrics.GetOrRegisterGauge("signature_request_duration", nil),
```

Comment:
nit: can we rename either the variable or metric name?

Reply:
Ya I'll rename the variable

Reply:
Renamed